### DISCUSSION TITLE
Disable update on expand?

### DISCUSSION THREAD

**Discussion by hanshafner** [aonez/Keka#1237] [General]
Hi there,

I really appreciate the frequent updates to the app. It's good to know that Keka is being actively developed.

I wish though, that I could expand something and get the update notification when the expand is done... it is a little distracting to have to think about an update when I really want to expand something and work on that thing, but now I have to think about Keka...

It's a difficult problem to solve, I know...

**Comment by aonez** [comment]
Hi @hanshafner!

Totally understandable. Even if the update notification came after the operation you'll be probably bothered anyway since you'll be using the extracted/compressed files.

If you want Keka to be updated but not be bothered you can let it update by itself by setting `SUAutomaticallyUpdate` to `true`:
```sh
defaults write com.aone.keka SUAutomaticallyUpdate true
```

Probably I should add a check for the automatic installation too, since a it's really useful option to have.

You can also disable the automatic update checks in the Preferences and check for them manually when you have the time for it. But usually this is the worst idea since it's rare find the time for those tasks 😛

**Comment by hanshafner** [comment]
Thank you!

I set the automatic update to true.

I just had to vent a little bit... because I have 3 computers I feel every time I open Keka I'm getting that notice... :-)

**Comment by gingerbeardman** [comment]
This also bothers me so thanks for the workaround.
